include/mapbox/geometry/point.hpp

```diff
@@ -19,12 +19,17 @@ struct point
     T y;
 };
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wfloat-equal"
```

Comment:
Curious why ignored "-Wfloat-equal"?

Reply:
If we do not, then compare operations on any base class using a `double` or `float` type would have a compile warning and fail. This is a known issue that is being considered in #49. Until we figure out a great way to resolve this, I don't want to cause compilation failures.